packages/svelteui-core/src/components/Breadcrumb/BreadcrumbItem/BreadcrumbItem.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import useStyles from './BreadcrumbItem.styles';
+	import { Box } from '../../Box'; 
+	import type { BreadcrumbItemProps as $$BreadcrumbItemProps } from './BreadcrumbItem';
+	import type { BreadcrumbContext } from '../Breadcrumb';
+	import { getContext } from 'svelte';
+	import { ctx } from '../Breadcrumb.svelte';
+
+	interface $$Props extends $$BreadcrumbItemProps {}
+
+	export let use: $$Props['use'] = [],
+		element: $$Props['element'] = undefined,
+		className: $$Props['className'] = '',
+		override: $$Props['override'] = {},
+		color: $$Props['color'] = undefined,
+		href: $$Props['href'] = undefined,
+		size: $$Props['size'] = 'md',
+		active: $$Props['active'] = false;
+
+	export { className as class };
+
+	// retrieves the reactive context so that TimelineItem has access
+	// to the Timeline parameters
+	const state: BreadcrumbContext = getContext(ctx);
+
+	let separator = $state.separator;
+	size = $state.size;
+	color = $state.color;
```

Comment:
if a size and color are provided we want to use those instead of what comes from the parent so in this case it would be
```
size = size | $state.size;
color = color | $state.color;
```

Reply:
I have remove the input for breadcrumb item